unlaunchio/client/offline_test.go

```diff
@@ -0,0 +1,30 @@
+package client
+
+import (
+	"testing"
+	"github.com/unlaunch/go-sdk/unlaunchio/util/logger"
+)
+
+
+func offlineClientWithMocks() *OfflineClient {
+	mfs.ready = true
+
+	return &OfflineClient{
+		FeatureStore:          mfs,
+		eventsRecorder:        nil,
+		eventsCountAggregator: nil,
+		logger:                logger.NewLogger(nil),
+		evaluator:             &mockEvaluator{},
+	}
+}
+
+func TestWhen_OfflineMode(t *testing.T) {
```

Comment:
Need test for "Feature()" method as well. You should confirm that Variation is "control" and that Evaluation Reason is also what you provided.